mllib-local/src/main/scala/org/apache/spark/ml/linalg/Matrices.scala

```diff
@@ -759,6 +769,8 @@ object SparseMatrix {
 
 /**
  * Factory methods for [[org.apache.spark.ml.linalg.Matrix]].
+ *
+ * @since 2.0
```

Comment:
@srowen Jira description says that these can not be annotations:

> In spark-mllib-local, we're no longer to be able to use @Since annotation. As a result, we will switch to standard java doc style using /* @Since /*. This task will add those new APIs as @Since 2.0

Not sure of reasoning behind it though.